src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/PublishContent.targets

```diff
@@ -25,7 +25,9 @@
     <CreateAzureContainer
       AccountKey="$(CloudDropAccessToken)"
       AccountName="$(CloudDropAccountName)"
-      ContainerName="$(ContainerName)" />
+      ContainerName="$(ContainerName)"
+      WriteOnlyTokenDaysValid="1"
+      ReadOnlyTokenDaysValid="1" />
```

Comment:
If you do leave this in, @karajas please be sure to back it out when you fix the handling of zero.